libraries/AP_HAL_ChibiOS/hwdef/speedybeef4v3/hwdef.dat

```diff
@@ -152,8 +152,9 @@ define HAL_OSD_TYPE_DEFAULT 1
 ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
 
 # minimal drivers to reduce flash usage
-include ../include/minimal.inc
+include ../include/minimize_fpv_osd.inc
 include ../include/no_bootloader_DFU.inc
-define AP_BATTERY_SYNTHETIC_CURRENT_ENABLED 0
 
 define DEFAULT_NTF_LED_TYPES 257
+
+AUTOBUILD_TARGETS Copter
```

Comment:
Don't think you should do this. I know there is a wing version, but its a fairly standard 30x30 setup

Reply:
you're right....thought there was a SpeedyBee stack that you recently did that on and was just following suit, but this is offered standalone and has 8 pwm...will change

Reply:
done